<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_edit_form</class>
 <widget class="QDialog" name="add_edit_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>335</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>ID: </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_id">
     <property name="text">
      <string>00</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <widget class="QPushButton" name="button_ok">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Название сорта кофе:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" rowspan="2">
    <widget class="QPushButton" name="button_cancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="sort"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Степень прожарки:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Цена, руб:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox_proj"/>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="price"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Степень помола:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Вес упаковки, гр:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QComboBox" name="comboBox_pomol"/>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="vol"/>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Описание вкуса:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="4">
    <widget class="QTextEdit" name="vkus"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
